[Bug] Basic benchmark test doesnt run through

The benchmark test dont run through locally. I get the error:

`FAILED tests/test_benchmark.py::test_run_smoothly_csv - FileExistsError: Output folder .../mvs_eland/tests/MVS_outputs_simulation already exists. If you want to overwrite the folder, please chosse overwrite ...`

_Originally posted by @Piranias in https://github.com/rl-institut/mvs_eland/pull/259#issuecomment-614576506_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -53,6 +53,7 @@ tipps for module building, and hint that units in the MVS are not checked (PR #2
 - Fix plotting error in F1, plot only if Data frame is not empty (#230, #234)
 - Fix naming error for storages (#166)
 - Benchmark test that the simulation is running with default settings (#254)
+- Overwrite local results when running through brenchmark tests (#260)
 
 ## [0.2.0] - 2020-03-13
 

--- a/tests/test_benchmark.py
+++ b/tests/test_benchmark.py
@@ -42,7 +42,7 @@ def test_run_smoothly_json(mock_args):
 )
 @mock.patch("argparse.ArgumentParser.parse_args", return_value=argparse.Namespace())
 def test_run_smoothly_csv(mock_args):
-    main(input_type=CSV_EXT, path_output_folder=OUTPUT_PATH)
+    main(input_type=CSV_EXT, path_output_folder=OUTPUT_PATH, overwrite=True)
     assert 1 == 1
 
 

